tools/wptrunner/wptrunner/executors/executorselenium.py

```diff
@@ -155,7 +155,7 @@ def test_window_loaded(self):
         """
         while True:
             try:
-                self.webdriver.execute_script(self.window_loaded_script, asynchronous=True)
+                self.webdriver.execute_async_script(self.window_loaded_script)
```

Comment:
Note: we need to figure out why this works today; it isn't obviously Py3 specific. I am currently looking into this (unless you have any knowledge here Ziran).

The general gist is that `self.webdriver` here is a `selenium.Remote`, which doesn't have an `execute_script` method with an `asynchronous` parameter. So this *should* fail in Py2 today, but appears not to!

Reply:
Does this not run for the Edge Stable bots?

Reply:
soab, we've been running 'edge' instead of 'edgechromium' in our testing -_-

Reply:
@ziransun I'm sorry, this is on me. I totally forgot that 'edge' on WPT isn't the new edge, it's the old EdgeHTML. New edge is called 'edgechromium' for WPT.

Can you restart your efforts using 'edgechromium' and see what changes we need?

Reply:
IIRC, we've transitioned: Edge via Selenium -> Edge via EdgeDriver (originally `edge_webdriver`, but later became the default `edge` product) -> Edge Chromium. So the Selenium code path probably hasn't been used on any bot for quite a while.

Reply:
I see :). Yes, I will start using "edgechromium" instead. Thank you!

Reply:
> Can you restart your efforts using 'edgechromium' and see what changes we need?

@stephenmcgruer: Just tried with 'edgechromium' using Edge browser (version 84.0.522.49), I'm able to run the sample test dom/historical.html in python 3. Would it make sense to do a full windows run like we did in #23182?

Reply:
@ziransun - unfortunately, https://github.com/web-platform-tests/wpt/issues/24846 is blocking this :(

In the meantime, can you try running a few other tests locally, particularly one that relies on a Python file hander? Thanks!

Reply:
@ziransun I managed to get a run started, but quickly (before a single test was run) hit an error which looks to be Py3 related - https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=52119&view=logs&j=2b376673-b6b7-55d3-67e2-e8305abf5c1e&t=74a04147-a139-56ed-eca1-af09b5915baa

```
2020-08-03T20:01:48.9462273Z Traceback (most recent call last):
2020-08-03T20:01:48.9464940Z   File "./wpt", line 21, in <module>
2020-08-03T20:01:48.9465611Z     wpt.main()
2020-08-03T20:01:48.9466285Z   File "D:\a\1\s\tools\wpt\wpt.py", line 171, in main
2020-08-03T20:01:48.9466852Z     rv = script(*args, **kwargs)
2020-08-03T20:01:48.9467436Z   File "D:\a\1\s\tools\wpt\run.py", line 820, in run
2020-08-03T20:01:48.9468064Z     rv = run_single(venv, **wptrunner_kwargs) > 0
2020-08-03T20:01:48.9468678Z   File "D:\a\1\s\tools\wpt\run.py", line 827, in run_single
2020-08-03T20:01:48.9470601Z     return wptrunner.start(**kwargs)
2020-08-03T20:01:48.9471234Z   File "D:\a\1\s\tools\wptrunner\wptrunner\wptrunner.py", line 397, in start
2020-08-03T20:01:48.9471954Z     rv = not run_tests(**kwargs) or logged_critical.has_log
2020-08-03T20:01:48.9472555Z   File "D:\a\1\s\tools\wptrunner\wptrunner\wptrunner.py", line 187, in run_tests
2020-08-03T20:01:48.9473021Z     **kwargs)
2020-08-03T20:01:48.9473599Z   File "D:\a\1\s\tools\wptrunner\wptrunner\wptrunner.py", line 59, in get_loader
2020-08-03T20:01:48.9474459Z     enable_webrender=kwargs.get("enable_webrender"))
2020-08-03T20:01:48.9475051Z   File "D:\a\1\s\tools\wptrunner\wptrunner\wpttest.py", line 83, in get_run_info
2020-08-03T20:01:48.9475664Z     return RunInfo(metadata_root, product, **kwargs)
2020-08-03T20:01:48.9476239Z   File "D:\a\1\s\tools\wptrunner\wptrunner\wpttest.py", line 100, in __init__
2020-08-03T20:01:48.9476788Z     rev = GitTree(log_error=False).rev
2020-08-03T20:01:48.9477384Z   File "D:\a\1\s\tools\wptrunner\wptrunner\update\tree.py", line 174, in rev
2020-08-03T20:01:48.9477949Z     if vcs.is_git_root(self.root):
2020-08-03T20:01:48.9478496Z   File "D:\a\1\s\tools\wptrunner\wptrunner\vcs.py", line 57, in is_git_root
2020-08-03T20:01:48.9479131Z     rv = git("rev-parse", "--show-cdup", repo=path, log_error=log_error)
2020-08-03T20:01:48.9479730Z   File "D:\a\1\s\tools\wptrunner\wptrunner\vcs.py", line 36, in inner
2020-08-03T20:01:48.9480333Z     return func(command_line, stderr=subprocess.STDOUT, **proc_kwargs)
2020-08-03T20:01:48.9480974Z   File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\subprocess.py", line 356, in check_output
2020-08-03T20:01:48.9481551Z     **kwargs).stdout
2020-08-03T20:01:48.9482115Z   File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\subprocess.py", line 423, in run
2020-08-03T20:01:48.9485343Z     with Popen(*popenargs, **kwargs) as process:
2020-08-03T20:01:48.9486008Z   File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\subprocess.py", line 729, in __init__
2020-08-03T20:01:48.9486598Z     restore_signals, start_new_session)
2020-08-03T20:01:48.9487168Z   File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\subprocess.py", line 1017, in _execute_child
2020-08-03T20:01:48.9487710Z     startupinfo)
2020-08-03T20:01:48.9488147Z TypeError: CreateProcess() argument 8 must be str or None, not bytes
```